tuple/version.h

```diff
@@ -0,0 +1,10 @@
+#ifndef TUPLE_KEYDEF_VERSION_H_INCLUDED
+#define TUPLE_KEYDEF_VERSION_H_INCLUDED
+/*
+* Сontains the module version.
+* Requires manual update in case of release commit.
+*/
+
+static const char *_VERSION = "0.0.3";
```

Comment:
Several nits:

* Let's set 0.0.4. I plan to release it soon.
* Let's use `#define` instead of `static const char *`. Having different same named variables (each in its own compilation unit) seems confusing. (We have one compilation unit here, but anyway.)
* Let's name the file `tuple/keydef_version.h` or `tuple/keydef/version.h`. Otherwise it looks like a `tuple` module version.
* Let's name the constant `TUPLE_KEYDEF_VERSION`. Just `VERSION` (or `_VERSION`) is too common.

Please, reflect it in https://github.com/tarantool/tuple-merger/pull/35 as well.

Reply:
Done